<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(50,50,50);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pgLogin">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>191</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lnClientName">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
         </property>
         <property name="placeholderText">
          <string>Введите имя пользователя</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255,255,255);
background-color: rgb(120,240,100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pgMessanger">
     <widget class="QPushButton" name="btnExit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>22</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255,255,255);
background-color: rgb(240,120,100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>591</width>
        <height>491</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="lblStatus">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255,255,255);</string>
             </property>
             <property name="text">
              <string>Статус</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Available</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Away</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Busy</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="lblDestination">
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255,255,255);</string>
             </property>
             <property name="text">
              <string>Получатель</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cmbDestination">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
             </property>
             <item>
              <property name="text">
               <string>None</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>All</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="lstMessages">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(100, 100, 100);
border-radius: 9px;               </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="lnMessage">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
           </property>
           <property name="placeholderText">
            <string>Введите сообщение...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSend">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
           </property>
           <property name="text">
            <string>Отправить</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnSendFile">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255,255,255);
background-color: rgb(100, 100, 100);
border-radius: 9px;               
border: 2px solid rgb(100, 100, 100);</string>
           </property>
           <property name="text">
            <string>&gt;..</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>610</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="styleSheet">
     <string notr="true">color: rgb(255,255,255);</string>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">color: rgb(255,255,255);</string>
   </property>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
